README.md

```diff
@@ -26,6 +26,26 @@ wget -O /usr/local/bin/ee https://raw.githubusercontent.com/EasyEngine/easyengin
 chmod +x /usr/local/bin/ee
 ```
 
+### Tab completions
+
+EasyEngine also comes with a tab completion script for Bash and ZSH. Just download [ee-completion.bash](https://raw.githubusercontent.com/EasyEngine/easyengine/develop-v4/utils/ee-completion.bash) and source it from `~/.bash_profile`:
+
+```bash
+source /FULL/PATH/TO/ee-completion.bash
+```
+
+Don't forget to run `source ~/.bash_profile` afterwards.
+
+If using zsh for your shell, you may need to load and start `bashcompinit` before sourcing. Put the following in your `.zshrc`:
+
+```bash
+autoload bashcompinit
+bashcompinit
+source /FULL/PATH/TO/ee-completion.bash
+```
+
+Note: Currently tab completions will only work for users with `root` priviledges.
```

Comment:
I think this is redundant as the whole EasyEngine software currently runs for users with superuser privileges.